sdk/core/core-amqp/review/core-amqp.api.md

```diff
@@ -393,12 +405,24 @@ export enum ErrorNameConditionMapper {
     UnauthorizedError = "amqp:unauthorized-access"
 }
 
+// @public
+export function isCredential(thing: unknown): thing is TokenCredential | NamedKeyCredential | SASCredential;
+
 // @public
 export function isMessagingError(error: Error | MessagingError): error is MessagingError;
 
+// @public
+export function isNamedKeyCredential(thing: unknown): thing is NamedKeyCredential;
+
+// @public
+export function isSASCredential(thing: unknown): thing is SASCredential;
+
```

Comment:
Why shouldnt these two live in core-auth?

Reply:
I don't have a good reason for that. I can add them there instead.

Reply:
Since isTokenCredential lives in core-auth, it makes sense for these two to move there as well

Reply:
Created PR here: https://github.com/Azure/azure-sdk-for-js/pull/14566